src/pages/Search/Search.tsx

```diff
@@ -1,64 +1,88 @@
-import React, { ChangeEvent, useEffect, useRef } from 'react';
-import { Search as SearchField } from '@equinor/eds-core-react';
+import React, { useState } from 'react';
+import { Button } from '@equinor/eds-core-react';
 import withAccessControl from '../../services/withAccessControl';
 import styled from 'styled-components';
-import useSearchPageFacade, { SearchStatus } from './useSearchPageFacade';
-import SearchResults from './SearchResults/SearchResults';
 import Navbar from '../../components/navigation/Navbar';
-import PageHeader from '../../components/PageHeader';
+import SearchArea from './SearchArea/SearchArea';
+import { COLORS } from '../../style/GlobalStyles';
 
 const SearchPageWrapper = styled.main`
     padding: 0 4%;
-    & h4 {
-        text-align: center;
-    }
-    & span {
-        height: 60px;
-        margin-bottom: 10px;
+`;
+
+const ButtonsWrapper = styled.div`
+    margin-bottom: 10px;
+    display: flex;
+    height: 60px;
+    & > button:not(:last-child) {
+        margin-right: 10px;
     }
 `;
 
+const SearchTypeButton = styled(Button)<{ active: boolean }>`
+    background-color: ${(props): string =>
+        props.active ? COLORS.fadedBlue : COLORS.white};
+    flex: 1;
+    height: 100%;
+`;
+
+export enum SearchType {
+    SAVED = 'SAVED',
+    PO = 'PO',
+    MC = 'MC',
+    WO = 'WO',
+    Tag = 'Tag',
+}
+
 const Search = (): JSX.Element => {
-    const { hits, searchStatus, query, setQuery } = useSearchPageFacade();
-    const searchbarRef = useRef<HTMLInputElement>(
-        document.createElement('input')
+    const [searchType, setSearchType] = useState<SearchType | undefined>(
+        SearchType.SAVED
     );
 
-    useEffect(() => {
-        searchbarRef.current?.focus();
-    }, []);
+    const buttonsToRender: JSX.Element[] = [];
 
-    const searchHeaderToRender = (): JSX.Element => {
-        if (searchStatus === SearchStatus.SUCCESS) {
-            return (
-                <h4>
-                    Displaying {hits.items.length} out of {hits.maxAvailable}{' '}
-                    packages
-                </h4>
+    for (const type in SearchType) {
```

Comment:
Denne for-løkka vil kjøre hver gang komponenten re-rendres. Hvis du ønsker å beholde det slik, bør denne legges inn i en funksjon. Lag en useEffect som kjører funksjonen når currentSearchType oppdateres.

MEN: Hva tenker du om å heller lage en "SearchTypeButton"-komponent som tar inn en "active"-prop, en "setCurrentSearchType"-prop og en "label"-prop, og rett og slett liste opp de fire knappene i stedet for å ha en for-loop?